bqetl_project.yaml

```diff
@@ -197,6 +197,8 @@ dry_run:
   - sql/moz-fx-data-shared-prod/org_mozilla_tiktokreporter/**/*.sql
   - sql/moz-fx-data-shared-prod/org_mozilla_ios_tiktok_reporter_tiktok_reportershare/**/*.sql
   - sql/moz-fx-data-shared-prod/org_mozilla_ios_tiktok_reporter/**/*.sql
+  - sql/moz-fx-data-shared-prod/fenix_derived/ltv_state_values_v1/query.sql
+  - sql/moz-fx-data-shared-prod/fenix_derived/ltv_state_values_v2/query.sql
```

Comment:
As discussed in the Jira ticket, the dryrun service account won't have access to `analysis`. These are the only two datasets atm that access an `analysis` dataset. They'll get productionized soon, so until then skip dryruns.